src/plugins/keywordNotify/index.tsx

```diff
@@ -0,0 +1,271 @@
+        matches.length ? (<span class="highlight">{matches.splice(0, 1)[0]}</span>) : []
+    ]);
+}
+
+const settings = definePluginSettings({
+    replace: {
+        type: OptionType.COMPONENT,
+        description: "",
+        component: () => {
+            const update = useForceUpdater();
+            const [values, setValues] = useState(regexes);
+
+            const elements = regexes.map((a, i) => {
+                const setValue = (v: string) => {
+                    let valuesCopy = [...values];
+                    valuesCopy[i] = v;
+                    setValues(valuesCopy);
+                }
+
+                return (
+                    <>
+                        <Forms.FormTitle tag="h4">Keyword Regex {i + 1}</Forms.FormTitle>
+
+                        <Flex flexDirection="row">
+                            <div style={{flexGrow: 1}}>
+                                <TextInput
+                                    placeholder="example|regex"
+                                    spellCheck={false}
+                                    value={values[i]}
+                                    onChange={setValue}
+                                    onBlur={() => setRegexes(i, values[i])}
+                                />
+                            </div>
+                            <Button
+                                onClick={() => removeRegex(i, update)}
+                                look={Button.Looks.BLANK}
+                                size={Button.Sizes.ICON}
+                                className="keywordnotify-delete">
+                                <DeleteIcon />
+                            </Button>
+                        </Flex>
+                    </>
+                )
+            });
+
+            return (
+                <>
+                    {elements}
+                    <div><Button onClick={() => addRegex(update)}>Add Regex</Button></div>
+                </>
+            );
+        }
+    },
+});
+
+export default definePlugin({
+    name: "KeywordNotify",
+    authors: [Devs.camila314],
+    description: "Sends a notification if a given message matches certain keywords or regexes",
+    settings,
+    patches: [
+        {
+    	    find: "}_dispatch(",
+    	    replacement: {
+                match: /}_dispatch\((\i),\i\){/,
+                replace: "$&$1=$self.modify($1);"
+    	    }
+    	},
+        {
+            find: "Messages.UNREADS_TAB_LABEL}",
+            replacement: {
+                match: /\i\?\(0,\i\.jsxs\)\(\i\.TabBar\.Item/,
+                replace: "$self.keywordTabBar(),$&"
+            }
+        },
+        {
+            find: "InboxTab.TODOS?(",
+            replacement: {
+                match: /:\i&&(\i)===\i\.InboxTab\.TODOS.{1,50}setTab:(\i),onJump:(\i),closePopout:(\i)/,
+                replace: ": $1 === 5 ? $self.tryKeywordMenu($2, $3, $4) $&"
+            }
+        },
+        {
+            find: ".default.guildFilter:null",
```

Comment:
```suggestion
            find: ".guildFilter:null,",
```
`default` can change